portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data;
+
+public final class Comment {
+  
+  public static final String ENTITY_NAME_PARAM = "comment";
+  public static final String TEXT_PARAM = "text";
+  public static final String TIME_PARAM = "timestamp";
+
+  private final String text;
+  private final long timestamp;
```

Comment:
It this timestamp in milliseconds or microseconds? Consider updating the variable names so they indicate the unit.

ex:
timestampMillis

Reply:
Done.